common/test/acceptance/tests/lms/test_oauth2.py

```diff
@@ -42,9 +42,9 @@ def test_cancelling_redirects(self):
         assert self.oauth_page.visit()
         self.oauth_page.cancel()
 
-        # This redirects to an invalid URI.
-        query = self._qs(self.browser.current_url)
-        self.assertEqual('access_denied', query['error'])
+        # This redirects to an invalid URI. current_url for firefox and title for chrome
+        query = self._qs(self.browser.current_url) or self._qs(self.browser.title)
```

Comment:
Please split this out into an if/else block that is based on browser.name. That will make the code easier for future developers to understand and maintain.
Something like:
```
        if self.browser.name == 'chrome':
            query = self._qs(self.browser.title)
        else:
            query = self._qs(self.browser.current_url)
```